packages/fgtclb/academic-contact4pages/Configuration/TypoScript/Page/Plugin.typoscript

```diff
@@ -8,6 +8,7 @@ plugin.tx_academiccontacts4pages {
     }
 
     partialRootPaths {
+      5 = EXT:academic_persons/Resources/Private/Partials/
```

Comment:
This is only okay because `EXT:contact4pages` already hard requires `EXT:academic_persons`.

I would still love to see a additional comment here stating that this is bound to that requirement !!!!